compiler/testData/codegen/boxAgainstJava/visibility/protectedAndPackage/protectedPropertyInPackageFromCrossinline.kt

```diff
@@ -0,0 +1,19 @@
+// FILE: protectedPack/J.java
+
+package protectedPack;
+
+public class J {
+    protected String foo = "OK";
+}
+
+// FILE: 1.kt
+
+package protectedPack
+
+inline fun foo(crossinline bar: () -> String) = object {
+    fun baz() = bar()
+}.baz()
+
+fun box(): String {
```

Comment:
Could you add more test with protected, e.g. when protected methods are called from crossinline lambda, but in case when it's inlined again in inline function, e.g.:
class A {
  protected fun smth() {}
}

class B: A() {
   inline fun test(l: () -> ?) = inlineCrossInlineCall{ foo(); l() }
}

and test call from another package

Reply:
Done. This is actually an interesting example, as it fails on JVM (while JVM_IR correctly traces the call site to an inline function and adds an accessor).